mpv-plugin/open-in-mpv/open-in-mpv-2.4.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module desktop xdg-utils
```

Comment:
@vitaly-zdanevich You can use simply `xdg`, and remove the pkg_postinst() and postrm()